web/src/assets/i18n/gb.json

```diff
@@ -129,7 +129,7 @@
 	"restoreWindow": "Restore window",
 	"restoreZoom": "Restore zoom",
 	"save": "Save",
-	"saveBeforeALMExport": "Save before ALM export?",
+	"saveBeforeTestprocedureExport": "Save before testprocedure export?",
```

Comment:
um konsistent zu oben zu sein: Before the export the test procedure will be saved. Continue?